src/Query/TaskGroups.ts

```diff
@@ -60,7 +63,7 @@ export class TaskGroups {
      *
      * Note that tasks may be displayed more than once. For example any tasks with more than one
      * tag will be displayed multiple times if grouped by Tag.
-     * So summing the number of tasks in all the {@link groups} can give a larger number than
+     * So summing the number of tasks in all the {@link _groups} can give a larger number than
```

Comment:
The previous version was better as it linked to the public getter, rather than now where it links to the private field.

Reply:
It was giving me errors on VSCode somehow, so I forgot about the getter. I restored the previous state and it seems fine now O_o